src/lib.rs

```diff
@@ -0,0 +1,72 @@
+//! Simple lock initially backed by PostgreSQL.
+mod errors;
+
+#[cfg(feature = "postgres")]
+pub mod postgres;
+
+use std::fmt::Debug;
```

Comment:
```suggestion
```

This import is not necessary fwiw

Reply:
the compiler threw a fit about it because I was including it as a trait on the `LockClient`